packages/blockchain-wallet-v4-frontend/src/components/Rows/OptionRightAction.tsx

```diff
@@ -55,7 +91,8 @@ const OptionRightActionRow = ({ children, disabled, onClick, toolTip }: Props) =
           </TooltipHost>
         </div>
       ) : (
-        <Icon name='chevron-right' size='25px' color='grey400' />
+        // @ts-ignore grey400
+        <Icon name='chevron-right' color='#98A1B2' size='md' />
```

Comment:
why use `ts-ignore`?

Reply:
workaround for constellation stuff. it's fixed now.